### ISSUE TITLE
AI Assistant: Use base64 for images so private blogs can still use the image extension

### ISSUE THREAD

**Issue by dhasilva** [Automattic/jetpack#42208]
Right now we send the image URL for the API, but if the image is not publicly accessible, like on internal sites, the alt text and caption generations fail. To bypass that, we can send the image as base64 instead.
